src/components/Dashboard/DashboardCard/DashboardCardActionsBody.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const DashboardCardActionsBody = ({ children }) => <div className="kubevirt-dropdown__items">{children}</div>;
```

Comment:
className should be renamed to reflect the name of the component ie `kubevirt-dashboard-actions__body` and it also shouldn't be in `dropdown.scss` as it represents generic actions which have nothing to do with dropdown